utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,144 @@
+        return {}
+
+
+def migrate_jsonb():
+    table = CONFIG["table"]
+    id_col = CONFIG["id_column"]
+    jcol   = CONFIG["json_column"]
+
+    select_sql = f"SELECT {id_col}, {jcol} FROM {table} WHERE {jcol} IS NOT NULL"
+
+    with psycopg2.connect(CONFIG["db"]["dsn"]) as conn:
+        with conn.cursor(cursor_factory=DictCursor) as cur:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)